src/Agent/NewRelic/Agent/Core/Samplers/GCSamplerV2.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#if NETSTANDARD
```

Comment:
Should we start using `!NETFRAMEWORK` to make our transition away from Standard easier? We can also just address it later.

Reply:
I count 22 `#if NETSTANDARD2_0` and 19 `#if !NETFRAMEWORK` usages currently. It sure wouldn't hurt if we were consistent everywhere, though.

It's been part of my regular routine to find and update these when updating the .NET 8 prototype branch from `main` so don't think it's a huge issue.